data/pools/transactionPool.go

```diff
@@ -65,6 +62,7 @@ type TransactionPool struct {
 	expiredTxCount         map[basics.Round]int
 	pendingBlockEvaluator  BlockEvaluator
 	numPendingWholeBlocks  basics.Round
+	feePerByte             atomic.Uint64
```

Comment:
Just wanted to add context by mentioning that according to https://pkg.go.dev/sync/atomic#pkg-note-BUG, types atomic.Int64 and atomic.Uint64 are automatically 64-bit aligned, so they can be placed anywhere in a struct definition